<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finishing_report</class>
 <widget class="QMainWindow" name="finishing_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../universal/OpenRPT/images/OpenRPTWrtembed.qrc">
    <normaloff>:/OpenRPT/icons/filed_32.png</normaloff>:/OpenRPT/icons/filed_32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Section:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Shift:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dtpStartDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dtpEndDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="cboShift">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Day</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Night</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cboSection">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gluing - H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gluing - M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stitching</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Peeling</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="cmdRefresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="txtData"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../universal/OpenRPT/images/OpenRPTWrtembed.qrc"/>
 </resources>
 <connections/>
</ui>
